Source/Core/DolphinWX/Main.cpp

```diff
@@ -288,6 +294,11 @@ void DolphinApp::AfterInit()
 	else if (m_load_file && !m_file_to_load.empty())
 	{
 		main_frame->BootGame(WxStrToStr(m_file_to_load));
+		if (m_load_state && !m_state.empty())
+		{
+			SConfig::GetInstance().bLoadSaveState = true;
+			SConfig::GetInstance().m_state = m_state;
```

Comment:
Does this handle non-ASCII characters correctly? You may need to use `WxStrToStr`.